support/cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/web/AbstractOAuth20Tests.java

```diff
@@ -176,6 +176,8 @@
     public static final String PASSWORD = "password";
     public static final String GOOD_USERNAME = "test";
     public static final String GOOD_PASSWORD = "test";
+    public static final String CODE_CHALLENGE = "myclientcode";
+    public static final String CODE_CHALLENGE_METHOD_PLAIN = "plain";
```

Comment:
Should be unnecessary; Constant is already available in OAuthConstants?

Reply:
These are test constants not available in OAuthConstants.